packages/worker-shared/src/userAssetUploads.ts

```diff
@@ -26,43 +29,93 @@ export async function handleUserAssetUpload({
 
 export async function handleUserAssetGet({ request, bucket, objectName, context }: UserAssetOpts) {
 	const cacheUrl = new URL(request.url)
+	const shouldOptimize =
+		request.query.tl_opt && !/image-resizing/.test(request.headers.get('via') ?? '')
+
+	let format = null
+	if (shouldOptimize) {
+		const accept = request.headers.get('Accept')
+		if (accept && /image\/avif/.test(accept)) {
```

Comment:
quick initial comment: why do this manually? Cloudflare is handling the `format=auto` depending on the request headers to do the right thing. we should keep the behavior there (mostly it returns avif)

Reply:
`format=auto` isn't supported by these worker-based transforms, this is what's suggested in the cloudflare docs, see the note at the bottom of this `format` section: https://developers.cloudflare.com/images/transform-images/transform-via-workers/#format